notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Copyright (c) Microsoft Corporation. All rights reserved.*\n",
+    "\n",
+    "*Licensed under the MIT License.*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Riemannian Low-rank Matrix Completion algorithm on Movielens dataset\n",
+    "\n",
+    "Riemannian Low-rank Matrix Completion (RLRMC) is a matrix factorization based (vanilla) matrix completion algorithm that solves the optimization problem using Riemannian conjugate gradients algorithm. \n",
+    "\n",
+    "This notebook provides an example of how to utilize and evaluate RLRMC implementation in **reco_utils**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import sys\n",
+    "import time\n",
+    "import pandas as pd\n",
+    "sys.path.append(\"../../\")\n",
+    "sys.path.append(\"../../reco_utils/recommender/rlrmc/\")\n",
+    "\n",
+    "from reco_utils.dataset.python_splitters import python_random_split\n",
+    "\n",
+    "from reco_utils.dataset import movielens\n",
+    "\n",
+    "from reco_utils.recommender.rlrmc.RLRMCdataset import RLRMCdataset \n",
+    "from reco_utils.recommender.rlrmc.RLRMCalgorithm import RLRMCalgorithm\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
```

Comment:
I love this!
```
%load_ext autoreload
%autoreload 2
```

Btw, would be cleaner and nicer if we separate them from the import cell. Also very tiny comment about the format: no spaces between the imports of the same group, e.g.
```
from reco_utils ...
from reco_utils ...
...
```

Reply:
Sorry I did not understand the comment regarding formatting. Please explain it in more details.

Reply:
done